doc/release-notes/release-notes-5.5.0-rc3.md

```diff
@@ -0,0 +1,467 @@
+Notable changes
+===============
+
+RPC Changes
+-----------
+
+- `getdeprecationinfo` has several changes:
+  - It now returns additional information about currently deprecated and
+    disabled features.
+  - A new `end_of_service` object that contains both the block height for
+    end-of-service and the estimated time that the end-of-service halt is
+    expected to occur. Note that this height is just an approximation and
```

Comment:
I think it should be “Note that this time…”

Reply:
```suggestion
    expected to occur. Note that this time is just an approximation and
```

Reply:
@r3ld3v Nice catch, thanks! Also, good to see you here. :)